main.py

```diff
@@ -43,5 +46,19 @@ def main(input_code, input_lang, output_lang):
     
     output_code = gtc.gast_to_code(gast, output_lang)
 
+    # Use a service account
+    cred = credentials.Certificate('firebase-acc.json')
+    firebase_admin.initialize_app(cred)
+
+    # Add user query and result to database
+    db = firestore.client()
+    doc_ref = db.collection(u'user-query').document()
+    doc_ref.set({
```

Comment:
Could this also have some flag or report if the translation failed? It would make it a lot easier to query if that computation is done before writing instead of when searching, especially if the error format changes over time.

Reply:
Also can we track if this is an `automatic translation` or a `manual translation`? That might be useful to query by.